tests/pass-dep/libc/pthread-threadname.rs

```diff
@@ -213,9 +213,15 @@ fn main() {
                     }
                 }
             };
-            let res = unsafe { libc::pthread_setname_np(invalid_thread, [0].as_ptr()) };
-            assert_eq!(res, error);
-            let res = unsafe { libc::pthread_getname_np(invalid_thread, [0; MAX_THREAD_NAME_LEN].as_mut_ptr(), MAX_THREAD_NAME_LEN) };
+            cfg_if::cfg_if!{
+                if #[cfg(not(target_os = "macos"))] {
+                    // macOS has no function accepting a thread id as the first argument.
+                    let res = unsafe { libc::pthread_setname_np(invalid_thread, [0].as_ptr()) };
+                    assert_eq!(res, error);
+                }
+            }
```

Comment:
No need to use `cfg_if` here, just use normal `#[cfg]` on a block. `cfg_if` only makes sense when we want an `else`.